bitcode2cpp.py

```diff
@@ -152,6 +138,3 @@
 else:
     sys.stderr.write("Unknown argument for --type: " + args[0].type + "\n")
     sys.exit(1)
-
-as_out.wait()
-sys.exit(as_out.returncode)
```

Comment:
I changed this script. It now takes input from the file and write output to the file. Previously, it operated with stdin/stdout. It is cleaner to use in cmake (e.g., not escape like this needed `| \"${Python3_EXECUTABLE}\"`) and it may allows us to resolve https://github.com/ispc/ispc/issues/2479 (write to temp file and then atomically rename it).

Reply:
I have added the fix to mentioned ticket as well.